etl/config.py

```diff
@@ -0,0 +1,2 @@
+username = 'steinar'
```

Comment:
This shouldn't be in the repo

Reply:
I will untrack this file in git once the .gitignore is updated